settings.py

```diff
@@ -88,9 +88,11 @@
 # This is the index that has the original text to be indexed. This is also used
 # when displaying the actual document from the search results. Must have
 # trailing slash. The user running the app must have read access to the path.
-# type: string
+# type: string | tuple
 # default: '%(sherlock_dir)s/tests/text/'
-INDEX_PATH = config.get('index_path', '%(sherlock_dir)s/tests/text/')
+INDEX_PATHS = config.get('index_path', '%(sherlock_dir)s/tests/text/' % {'sherlock_dir': ROOT_DIR})
+if isinstance(INDEX_PATHS, (str, unicode)):
```

Comment:
Yes, it will have to, at least on the other branch. Currently the dependency is not here though. I did not want to mix them up and add it here just for this.